streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -494,6 +494,7 @@ public void process(final ProcessorSupplier<K, V> processorSupplier, String... s
             .singleton(repartitionedSourceName), false);
     }
 
+    //TODO add reset strategy to repartitioned source as well?
```

Comment:
You absolutely correct, in this context "latest" would not make sense, removing TODO comment